source/common/common/key_value_store_base.cc

```diff
@@ -35,12 +39,20 @@ KeyValueStoreBase::KeyValueStoreBase(Event::Dispatcher& dispatcher,
     : max_entries_(max_entries), flush_timer_(dispatcher.createTimer([this, flush_interval]() {
         flush();
         flush_timer_->enableTimer(flush_interval);
-      })) {
+      })),
+      ttl_manager_([this](const auto& expired) { ttlCallback(expired); }, dispatcher,
```

Comment:
nit: const std::vector<std::string>& expired might be more clear than using auto